plutus-core/plutus-core/test/Parser/Spec.hs

```diff
@@ -20,7 +20,7 @@ import Test.Tasty.Hedgehog
 propTermSrcSpan :: Property
 propTermSrcSpan = property $ do
     term <- _progTerm <$>
-        forAllWith display (runAstGen $ discardIfAnyConstant (not . isSerialisable) genProgram)
+        forAllWith display (runAstGen $ regenConstantsUntil isSerialisable =<< genProgram)
```

Comment:
It's much faster to regenerate a specific unsupported constant than to discard the whole term and regenerate a new one from scratch.